scripts/upgrade/generate-upgrade-config.ts

```diff
@@ -4,10 +4,31 @@ import fs from "fs";
 import path from "path";
 import type { HardhatRuntimeEnvironment } from "hardhat/types";
 import { getSelectors } from "../libraries/diamond";
+import { getFunctionSignatureDetails } from "../libraries/metaTransaction";
 
 const prefix = "contracts/";
 const sources = ["diamond", "protocol/facets", "protocol/clients", "protocol/bases", "protocol/libs"];
 
+const RESTRICTED_FUNCTIONS = [
+  "burn",
+  "mint",
+  "renounceOwnership",
+  "transferOwnership",
+  "setTransferValidator",
+  "transferFractionsFrom",
+  "addPriceOracle",
+  "removePriceOracle",
+  "adjustVotesOnTransfer",
+  "migrateFractions",
+  "setMaxRoyaltyPercentage",
+  "setOpenSeaFeePercentage",
+  "setProtocolFeeTable",
```

Comment:
I suppose we should add all functions from `configFacet` here, not only the new ones. Probably you could just exclude the whole `configFacet` and similarly the `pauseFacet`.